specification/addons/resource-manager/Microsoft.Addons/preview/2017-05-15/Addons.json

```diff
@@ -0,0 +1,427 @@
+                }
+            }
+        },
+        "OperationsDisplayDefinition": {
+            "type": "object",
+            "description": "Display object with properties of the operation.",
+            "properties": {
+                "provider": {
+                    "type": "string",
+                    "description": "Resource provider of the operation."
+                },
+                "resource": {
+                    "type": "string",
+                    "description": "Resource for the operation."
+                },
+                "operation": {
+                    "type": "string",
+                    "description": "Short description of the operation."
+                },
+                "description": {
+                    "type": "string",
+                    "description": "Description of the operation."
+                }
+            }
+        },
+        "ErrorDefinition": {
+            "type": "object",
+            "title": "Error",
+            "description": "Error description and code explaining why an operation failed.",
+            "required": [
+                "message",
+                "code"
+            ],
+            "properties": {
+                "message": {
+                    "type": "string",
+                    "description": "Description of the error."
+                },
+                "code": {
+                    "type": "integer",
+                    "description": "Service specific error code which serves as the substatus for the HTTP error code.",
+                    "minimum": 100,
+                    "maximum": 600
+                }
+            }
+        }
+    },
+    "parameters": {
+        "SubscriptionIdParameter": {
+            "name": "subscriptionId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Subscription credentials that uniquely identify the Microsoft Azure subscription. The subscription ID forms part of the URI for every service call."
+        },
+        "ApiVersionParameter": {
+            "name": "api-version",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "Client API version."
+        },
+        "SupportProviderName": {
+            "name": "providerName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The support plan type. For now the only valid type is \"canonical\".",
+			"x-ms-parameter-location": "method"
+        },
+        "CanonicalSupportPlanType": {
+            "name": "planTypeName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The Canonical support plan type. Can be \"essential\", \"standard\" or \"advanced\".",
+            "enum": ["essential", "standard", "advanced"],
```

Comment:
spelling different from the one in your examples! Please adjust either examples or this `enum` to match what your service actually sends/expects (if it accepts both but sends one flavor, use the flavor that is being sent, etc...)